web/cypress/support/page_objects/onboardingPage.ts

```diff
@@ -83,7 +83,7 @@ export class OnboardingPage {
   }
 
   getLegalStructure(companyType: string) {
-    return cy.get(`[data-value="${companyType}"]`);
+    return cy.get(`input[name="legal-structure"][value="${companyType}"]`);
```

Comment:
do we need this function, if we never interact with legal structure on the onboarding page anymore?